app/routes/__index/certificate/index.tsx

```diff
@@ -85,13 +85,13 @@ export default function CertificateIndexRoute() {
     return date;
   }
 
-  if (certificate?.status === 'pending') {
-    return (
-      <Loading
-        img={pendingSvg}
-        desc="We have received your request, and will notify you when your certificate is ready"
-      />
-    );
+  function checkIfRenewValid(): boolean {
+    let currentDate = new Date();
+    let validTo = new Date(certificate.validTo!);
+    let thirtyDays = (30 * 24 * 60 * 60 * 1000) / (1000 * 60 * 60 * 24);
```

Comment:
Wouldn't it make sense to have this be 30?
Two sides of divide on `(24 * 60 * 60 * 1000) / (1000 * 60 * 60 * 24)` just cancel each other out.
In fact, I think we can probably use 30 instead of a variable.

Reply:
You can use dayjs for this too to simplify.

Reply:
oh great that makes it much easier